dnf/__init__.py

```diff
@@ -22,7 +22,7 @@
 import warnings
 import dnf.pycomp
 
-warnings.filterwarnings('once', category=DeprecationWarning)
+warnings.filterwarnings('once', category=DeprecationWarning, module='^dnf\..*$')
```

Comment:
I'm getting this warning here:
/home/mblaha/src/dnf/dnf/__init__.py:25: DeprecationWarning: invalid escape sequence \. warnings.filterwarnings('once', category=DeprecationWarning, module='^dnf\..*$')

Reply:
you probably need `warnings.filterwarnings('once', category=DeprecationWarning, module=r'^dnf\..*$')`